scraper/src/tests/default_strategy/get_records_from_dom_test.py

```diff
@@ -492,3 +492,34 @@ def test_objectID(self):
         assert actual[2]['objectID'] is not None
         assert actual[2][
                    'objectID'] == '71980d31d18995da99751933a79075e35b1a68bc'
+
+    def test_current_level(self):
+            # Given
+            strategy = get_strategy({
+                'selectors': {
+                    "lvl0": "h1",
+                    "lvl1": "h2",
+                    "lvl2": "h3",
+                    "content": "p"
+                },
+                'start_urls': [
+                    'http://test.com/docs/guides'
+                ]
+            })
+
+            html = """
+            <html><body>
+                <h1>Title</h1>
+                <h2>Chapter</h2>
+                <h3></h3>
+                <p>>Bim</p>
```

Comment:
Although it doesn't change the output of the test, that's probably what you meant:

```suggestion
                <p>Bim</p>
```